docs/plugin-config.md

```diff
@@ -43,7 +43,7 @@ Default: **empty string**
 > -----END CERTIFICATE-----
 > ```
 >
-> **Note**
+> [!NOTE]
```

Comment:
Is this a standalone note or a nested note? as it is not rendered correctly.

Reply:
Thanks! Updated.